packages/roosterjs-editor-types/lib/consts/DarkMode.ts

```diff
@@ -0,0 +1,10 @@
+export const DARK_MODE_DEFAULT_FORMAT = {
```

Comment:
The initial idea of roosterjs-editor-types package is not to generate any js code, everything here is just interfaces, types, const enums. However the const will generate some js code. I don't have strong objection, but still think we can move it to somewhere else, unless we believe it will be used somewhere else as well.